CMakeLists.txt

```diff
@@ -43,12 +43,20 @@ set(XTENSOR_SPARSE_HEADERS
     ${XTENSOR_SPARSE_INCLUDE_DIR}/xtensor-sparse/xcoo_scheme.hpp
     ${XTENSOR_SPARSE_INCLUDE_DIR}/xtensor-sparse/xcsf_scheme.hpp
     ${XTENSOR_SPARSE_INCLUDE_DIR}/xtensor-sparse/xcsr_scheme.hpp
+    ${XTENSOR_SPARSE_INCLUDE_DIR}/xtensor-sparse/xeval.hpp
```

Comment:
Maybe we should rename this as `xsparse_eval.hpp` ? We could do the same for xscalar and maybe the scheme files (although it's also fine to keep them as is since there is no file with a similar name in xtensor).

Reply:
Keep the name as defined in `xtensor` seems better to me. It means that we have adapted the version for `xsparse`.
And making all files start with `xsparse` will quickly become unreadable, no?

Reply:
Ok!